plugins/uefi-capsule/uefi_capsule.conf

```diff
@@ -1,5 +1,8 @@
 [uefi_capsule]
 
+# enable the low-level debugging of fwupdx64.efi to the FWUPDATE_DEBUG_LOG EFI variable
```

Comment:
Perhaps we want a warning here that this is going to fill up the NVRAM store more quickly and should only be enabled for working on an issue?

Reply:
Also we need to mention it's not used for CoD

Reply:
Something like this?